cli/src/pcluster/config/cluster_config.py

```diff
@@ -37,9 +39,10 @@ def __init__(self, validator_class: Validator, priority: int = 1, **kwargs):
 class Config(ABC):
     """Represent an abstract Configuration entity."""
 
-    def __init__(self):
+    def __init__(self, raise_on_error=False):
```

Comment:
I think we should remove the `raise_on_error` from the `Config` objects,
instead we can pass it to the `validate()` function and it will be propagated to all the validators.

It is an option related to the validators not to the config.

Reply:
Done